repo.lb

```diff
@@ -88,6 +88,7 @@ class DevicesCache(dict):
                      "at90", "attiny", "atmega",
                      "samd21",
                      "samg55",
+                     "samv70",
```

Comment:
Let's put all three SAMs in one line before this grows out of control ;-P

Reply:
Done